core/templates/pages/contact-page/contact-page.component.html

```diff
@@ -4,35 +4,35 @@ <h1 tabindex="0" [innerHTML]="'I18N_CONTACT_PAGE_HEADING' | translate"></h1>
 <background-banner></background-banner>
 <div class="oppia-page-card oppia-static-full-content oppia-static-content-contact-page-below-banner">
   <div class="oppia-static-card-full-content oppia-static-card-content-full-wide e2e-test-contact-page">
-    <h2 [innerHTML]="'I18N_CONTACT_PAGE_GREETING_HEADING' | translate"></h2>
+    <h2 [innerHTML]="'I18N_CONTACT_PAGE_GREETING_HEADING' | translate" tabindex="0"></h2>
 
     <div class="oppia-contact-page flex">
-      <div class="oppia-contact-page-content">
-        <img id="donate" src="assets/images/contact_page/Donate img.svg" alt="Donate">
+      <div class="oppia-contact-page-content" tabindex="0">
+        <img id="donate" src="assets/images/contact_page/Donate img.svg" alt="">
         <h2 [innerHTML]="'I18N_CONTACT_PAGE_DONATE_HEADING' | translate"></h2>
         <p [innerHTML]="'I18N_CONTACT_PAGE_DONATE_PARAGRAPH' | translate"></p>
         <a href="https://www.oppia.org/donate">
-          <button class="hover-button" [innerHTML]="'I18N_CONTACT_PAGE_DONATE_TODAY_BUTTON' | translate"></button>
+          <button tabindex="-1" class="hover-button" [innerHTML]="'I18N_CONTACT_PAGE_DONATE_TODAY_BUTTON' | translate"></button>
```

Comment:
```suggestion
          <button tabindex="-1" class="hover-button">{{ 'I18N_CONTACT_PAGE_DONATE_TODAY_BUTTON' | translate }}</button>
```
Use data interpolation for variables with non-HTML values, here and elsewhere